packages/idyll-layouts/src/blog/styles.js

```diff
@@ -71,6 +71,32 @@ input {
   max-height: 100vh;
 }
 
+.overlay-div {
+  position: relative;
+}
+
+.overlay-button {
+  position: absolute;
+  top: 0;
+  right: 0;
+  opacity: .2;
+  background-color: #E7E3D0;
+  background-image: url('https://files.gitter.im/idyll-lang/Lobby/Zkzj/quill-icon.png');
```

Comment:
We should update this link to something on our domain. You can add the file to the `static` folder in `idyll-docs` to include it in the next push to `idyll-lang.org`

Reply:
Mmk - done via #414

Reply:
Now available: https://idyll-lang.org/static/images/quill-icon.png